feathr_project/setup.py

```diff
@@ -34,6 +34,8 @@
         "python-snappy",
         "pyyaml",
         "Jinja2",
+        "google",
+        "google-api-python-client",
```

Comment:
You might want to put some versioning here (like larger than a certain version) to make the maintenance easier.